### PR TITLE
ar71xx: Add support for wdr7800v1

### PR DESCRIPTION
**Pull request by JoeyJiao**
The device is similar to WDR6500, but with QCA988X and 6 external antennas which similars to WDR7500 and without USB.

Signed-off-by: Joey Jiao <joeyjiaojg@163.com>

Thanks for your contribution to OpenWrt!

To help keep the codebase consistent and readable,
and to help people review your contribution,
we ask you to follow the rules you find in the wiki at this link
https://openwrt.org/submitting-patches

Please remove this message before posting the pull request.

### CODE REVIEW

**Comment by ptpt52** [comment]
没有说明刷机方法呀，是否只能串口刷机？
配置情况要详细列出。

**Comment by JoeyJiao** [comment]
Yes, the same as WDR6500, only flash breed from uart first

File: target/linux/ar71xx/base-files/lib/ar71xx.sh [hunk 1]
```diff
@@ -1256,6 +1259,12 @@ ar71xx_board_detect() {
 	*"TL-WDR6500 v2")
 		name="tl-wdr6500-v2"
 		;;
+	*"TL-WDR7800 v1")
+		name="tl-wdr7800-v1"
+		;;
+	*"TL-WDR7800 v1")
```
Comment (Leo-PL) [inline target/linux/ar71xx/base-files/lib/ar71xx.sh]: This entry is duplicated.

**Review by Leo-PL — COMMENTED** [review]
Also, please consider targetting ath79 first, which is set to replace ar71xx in future.

**Comment by Leo-PL** [comment]
Also, if this isn't clear enough, ar71xx target will eventually get dropped.

**Comment by JoeyJiao** [comment]
Replaced by https://github.com/openwrt/openwrt/pull/1516

**Pull request closed without merging**